composer.json

```diff
@@ -1,37 +1,45 @@
 {
-    "name": "yiisoft/_____",
+    "name": "yiisoft/auth-jwt",
     "type": "library",
-    "description": "_____",
-    "keywords": ["_____"],
+    "description": "Yii auth JWT method",
+    "keywords": [
+        "auth",
+        "jwt",
+        "middleware",
+        "psr-15"
+    ],
     "homepage": "http://www.yiiframework.com/",
     "license": "BSD-3-Clause",
     "support": {
-        "issues": "https://github.com/yiisoft/_____/issues?state=open",
+        "issues": "https://github.com/yiisoft/auth-jwt/issues?state=open",
         "forum": "http://www.yiiframework.com/forum/",
         "wiki": "http://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii",
-        "source": "https://github.com/yiisoft/_____"
+        "source": "https://github.com/yiisoft/auth-jwt"
     },
     "require": {
-        "php": "^7.2"
+        "php": "^7.4|^8.0",
+        "yiisoft/auth": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.2",
-        "phan/phan": "^2.2 || ^3.0"
+        "infection/infection": "^0.17.0",
+        "phpunit/phpunit": "^9.3",
+        "nyholm/psr7": "^1.0",
```

Comment:
Is this requirement relevant now?

Reply:
Did you mean to replace it with httpsoft?

Reply:
Not. Why add a dependency if it's not used in this code?

Reply:
This will be used for tests

Reply:
OK. Let's remove it for now.